Gemfile

```diff
@@ -55,10 +55,10 @@ group :production do
 end
 
 group :development do
-  gem 'capistrano', '3.4.0'
-  gem 'capistrano-bundler', '1.1.4'
+  gem 'capistrano', '3.8.1'
+  gem 'capistrano-bundler', '1.2.0'
   gem 'capistrano-passenger', '0.2.0'
-  gem 'capistrano-rails', '1.1.3'
+  gem 'capistrano-rails', '1.2.3'
   gem 'capistrano-maintenance', '~> 1.0', require: false
```

Comment:
Why are we locking to specific versions of cap in the gemfile?  I'd do more like `gem "capistrano", "~> 3.8", meaning any 3.x greater than or equal to 3.8.x.